terraform/module-prometheus/outputs.tf

```diff
@@ -13,35 +13,36 @@
 #}
 
 output "prometheus_eip" {
-  value = "${element(aws_eip.prometheus.*.public_ip, 0)}"
+  value = element(aws_eip.prometheus.*.public_ip, 0)
 }
 
 output "prometheus_secgroup_id" {
-  value = "${aws_security_group.prometheus.id}"
+  value = aws_security_group.prometheus.id
 }
 
 output "prometheus_instance_id" {
-  value = "${aws_instance.prometheus.id}"
+  value = aws_instance.prometheus[0].id
 }
 
 # workaround to handle count = 0 on output
 # https://github.com/hashicorp/terraform/issues/16726
 output "rds_address" {
-  value = "${element(concat(aws_db_instance.grafana.*.address, list("")), 0)}"
+  value = element(concat(aws_db_instance.grafana.*.address, [""]), 0)
```

Comment:
I think it is not needed anymore. You can remove the # workaround comment and use 

```
output "rds_address" {
  value = aws_db_instance.grafana[0].address
}
```

Reply:
Will this work if `count = 0` ?

Reply:
Yes I tested it on lemp